_maps/shuttles/supply_rebel.dmm

```diff
@@ -0,0 +1,47 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```

Comment:
Copy pasta much, but shuttle system still doesn't allow to have the same shuttle with different ids

Reply:
Then fix it you shouldnt have to spam new shuttles every time

Reply:
fixed